solution.cpp

```diff
@@ -1,23 +1,175 @@
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
- * Released under the [MIT License] (http://opensource.org/licenses/MIT)
- * ------------------------------------------------------------------------- */
+* Copyright &copy; 2015 (Henry Yu) <(henryu1996.csu.fullerton.edu)>
+*Released under the [MIT License] (http://opensource.org/licenses/MIT)
+* ------------------------------------------------------------------------- */
 
 /**
- * TODO: program description
+ * This program converts integers to Roman numerals.
  */
 
 // TODO: `#include`s and function prototypes
+#include <iostream>
+#include <string>
+
+using std::cout;
+using std::cin;
+using std::string;
+using std::endl;
 
 // ----------------------------------------------------------------------------
 
-int main() {
-    // TODO
+#include "stdafx.h"
```

Comment:
i've probably mentioned this to you in lab by now, but: `stdafx.h` is a windows specific thing.  i forget what it's for, but it's probably best to avoid it unless you know for sure that you need it.